Source/WebKit/UIProcess/gtk/ViewGestureControllerGtk.cpp

```diff
@@ -39,7 +39,11 @@ static const Seconds swipeMaxAnimationDuration = 400_ms;
 static const double swipeAnimationBaseVelocity = 0.002;
 
 // GTK divides all scroll deltas by 10, compensate for that
```

Comment:
This comment belongs inside the `#else` now, right?

Reply:
Yep.